### PR TITLE
Restyle video item

### PR DESCRIPTION
**Pull request by IceDBorn**
Restyle video item to mimic youtube's style

### CODE REVIEW

**Comment by IceDBorn** [comment]
Also, I would like to know why the uploader's image is fetchable only from the feed page. Is it a limitation of newpipe?

**Comment by FireMasterK** [comment]
Thanks, it looks quite similar now!

Any particular reason why you removed the uploader image beside the channel? I would like to keep it, since it is present on the android app.

> Also, I would like to know why the uploader's image is fetchable only from the feed page. Is it a limitation of newpipe?

Yes, but now that you mention it, I will probably have a look at seeing if it can be added in NewPipeExtractor so that Piped can use it.

**Comment by IceDBorn** [comment]
Should I just add it back for the feed page until NewPipeExtractor implements the avatar fetch?

**Comment by FireMasterK** [comment]
> Should I just add it back for the feed page until NewPipeExtractor implements the avatar fetch?

Why not! I'm a member of the NewPipe team, and I should be able to work on this myself :)

**Comment by IceDBorn** [comment]
@FireMasterK Didn't know that you were a part of the team. I'll add it back in a few minutes.

**Comment by FireMasterK** [comment]
image

One small issue, it's not centered to the left when there's no image. (Or, was this a design choice? :thinking: )

File: src/components/VideoItem.vue [hunk 1]
```diff
@@ -23,38 +23,43 @@
...
+                <router-link :to="video.url + '&listen=1'">
+                    <font-awesome-icon icon="headphones"></font-awesome-icon>
+                </router-link>
+            </div>
         </router-link>
 
-        <div :class="{ 'uk-align-left': !(video.views >= 0 || video.uploadedDate) }">
-            <div v-if="video.uploaderUrl && video.uploaderName && !hideChannel">
+        <div v-if="video.uploaderUrl && video.uploaderName && !hideChannel" style="display: flex; flex-flow: row; height: 15%">
+            <img :src="video.uploaderAvatar" loading="lazy" class="uk-border-circle"/>
```
Comment (FireMasterK) [inline src/components/VideoItem.vue:38]: A `v-if` or a null check could be added here.

Comment (IceDBorn) [reply src/components/VideoItem.vue:38]: I am going to add a v-if, this commit was not the final one.

**Comment by FireMasterK** [comment]
Feel free to take your time, no worries!

**Comment by IceDBorn** [comment]
I think that this is the final one!

image

**Comment by FireMasterK** [comment]
Thank you for this, this is exactly how I had imagined it to look like!

**Comment by shibattor** [comment]
It looks awesome, thank you!

**Comment by FireMasterK** [comment]
On a side note, the uploader avatar will be added in Trending, Search, and Related Videos soon!

**Comment by shibattor** [comment]
I have an unrelated question, what about the sound only icon? It seems to be missing from the search results.

**Comment by IceDBorn** [comment]
A lot of things are missing from the video item when it can't fetch info about that specific thing. That's why you can see views and time lf upload when you're viewing a channel page or a playlist. It's the API's fault.

**Comment by FireMasterK** [comment]
> I have an unrelated question, what about the sound only icon? It seems to be missing from the search results.

This is actually because they're not actually rendered by `VideoItem.vue` but should be, it has been a long-pending activity for me to add some kind of distinction for the two.

**Comment by IceDBorn** [comment]
@FireMasterK I'm going to redesign a lot of parts of the UI, I want to contact you in any platform if it's possible to share my ideas with you.

**Comment by FireMasterK** [comment]
> @FireMasterK I'm going to redesign a lot of parts of the UI, I want to contact you in any platform if it's possible to share my ideas with you.

My prefered platform is Matrix, you can find my matrix handle at https://github.com/TeamPiped/Piped#contact

**Comment by shibattor** [comment]
If you need any testing or additional feedback (in this or any other ticket), let me know!

**Pull request merged**